### PR TITLE
emagged crushers will now completely delimb players

### PR DESCRIPTION
**Pull request by Spacedong**
instead of having it just deal 1,000 damage or whatever bullshit it was, why wasn't this default anyway?

### CODE REVIEW

**Comment by Nebyab** [comment]
because reasons spacedong

good ones i'm sure

File: code/game/machinery/recycler.dm [hunk 1]
```diff
@@ -148,10 +148,9 @@ var/const/SAFETY_COOLDOWN = 100
 	else
 		playsound(src.loc, 'sound/effects/splat.ogg', 50, 1)
 
-	var/gib = 1
-	// By default, the emagged recycler will gib all non-carbons. (human simple animal mobs don't count)
+	// By default, the emagged recycler will gib both carbons and non-carbons. (human simple animal mobs don't count)
 	if(iscarbon(L))
-		gib = 0
+		L.gib()
 		if(L.stat == CONSCIOUS)
```
Comment (Gandalf2k15) [inline code/game/machinery/recycler.dm:153]: How does a gibbed player plan on screaming? Swappies.

**Comment by Spacedong** [comment]
dont merge yet one more commita nd it should be good

**Comment by Hackole** [comment]
What? No, half the fun of an emagged crusher was leaving mutilated bodies everywhere, what is the point of this?

File: code/game/machinery/recycler.dm [hunk 2]
```diff
@@ -148,10 +148,9 @@ var/const/SAFETY_COOLDOWN = 100
...
 			L.say("ARRRRRRRRRRRGH!!!")
 		add_blood(L)
```
Comment (Gandalf2k15) [inline code/game/machinery/recycler.dm]: Why would you add blood to a body thats about to be gibbed?? Just spawn a blood decal below!

**Comment by Gandalf2k15** [comment]
Hackole has a point, can you just make it remove all limbs apart from head and chest @spacedong? Tha would be cool.

**Comment by Spacedong** [comment]
ok, fixes and changelog is done, good to go

**Comment by Gandalf2k15** [comment]
code loooks good

**Comment by calibraptor** [comment]
+1

**Comment by McHatters** [comment]
while it sounds a lot more fun I don't really dig the idea of a quick and easily accessible mass gibber. All of the gibbing methods we have at the moment either take time to gib someone, take time to make, or cost a good amount of TCs. The idea that a traitor can just get one emag and get a bunch of crushers sounds a bit too OP in my opinion.

I am down with the idea that the crushers remove limbs and all, though. A little less destructive than gibbing, but the player isn't removed from the round for good. Also, add the gibbing effect to the crusher even if it doesn't gib the guy.

**Comment by JohnGinnane** [comment]
Added feature as this will be a new thing. Added code review because I *think* people are suggesting making it detach the target's limbs (arms and legs)

**Comment by McBawbaggings** [comment]
Gibbing is boring, make it rip their limbs off and yank their organs in a bloody fashion.
Crusher should be stylish rather than ez gibs.

**Comment by Voratius** [comment]
delimbification sounds better than gibbing, aye

**Comment by dreamcrab** [comment]
yea i like making it explode you into a bunch of pieces that you have to pick up, plus the one time i was a cargo tech and just disposal dunked people with the crusher makes me think permadeath crusher is not a good thing

its really easy to dunk nerds

**Comment by Spacedong** [comment]
alright I'll make it delimb

**Comment by Spacedong** [comment]
don't merge this yet I have to get around to actually making it delimb the player

**Pull request closed without merging**